db/seeds.rb

```diff
@@ -5,3 +5,46 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+
+Member.create(
+	username:  'suee', 
```

Comment:
You might consider adding Rubocop to the project to help keep changes like this consistent across the project and multiple committers.